spec/rubocop/cop/performance/regexp_match_spec.rb

```diff
@@ -329,6 +329,24 @@ def foo
                     're =~ FOO', 'FOO.match?(re)')
     it_behaves_like(:all_legacy_match_methods, 'matching by =~`',
                     'FOO =~ re', 'FOO.match?(re)')
+    it_behaves_like(:all_legacy_match_methods, 'matching by !~`',
+      '/re/ !~ foo', '!/re/.match?(foo)')
```

Comment:
Offense of `Layout/AlignParameters` cop has occurred because these indentations are not aligned.

```diff
 it_behaves_like(:all_legacy_match_methods, 'matching by !~`',
-  '/re/ !~ foo', '!/re/.match?(foo)')
+                '/re/ !~ foo', '!/re/.match?(foo)')
```

https://travis-ci.org/bbatsov/rubocop/jobs/364429172#L605-L634

Reply:
Thanks, fixed!